packages/node-opcua-server/source/server_end_point.ts

```diff
@@ -510,8 +576,16 @@ export class OPCUAServerEndPoint extends EventEmitter implements ServerSecureCha
             debugLog("server is listening");
         });
 
+        const listenOptions: net.ListenOptions = {
+            port: this.port
+        };
+
+        if (this.host !== undefined && typeof this.host === "string" && this.host.length > 0) {
```

Comment:
host is string | undefined ,  undefined  resolves to false,  "" resolves to false too, therefore, we can simplify the code like this 
```
 if (this.host) { 
    listenOptions.host = this.host;
 }
```




which simplify further to

```
listenOptions.host = this.host;
``` 


which  can lead to 

```
        const listenOptions: net.ListenOptions = {
            port: this.port,
            host: this.host
        };
````